### PR TITLE
forgotten read_only options

### PR DESCRIPTION
**Pull request by axzx**

### CODE REVIEW

File: Tests/Form/Type/ModelAutocompleteTypeTest.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ public function testGetDefaultOptions()
         $this->assertSame(100, $options['quiet_millis']);
         $this->assertSame(false, $options['cache']);
         $this->assertSame('', $options['width']);
+        $this->assertSame(false, $options['read_only']);
```
Comment (greg0ire) [inline Tests/Form/Type/ModelAutocompleteTypeTest.php]: You could use `assertFalse`

**Comment by greg0ire** [comment]
Can you please [squash your commits](https://github.com/sonata-project/SonataAdminBundle/blob/master/CONTRIBUTING.md#squash-your-commits) ?

**Comment by axzx** [comment]
It is already well?

**Comment by greg0ire** [comment]
There are 2 commits, there should be only one. Rebase on the master branch of this repo to get rid of the first one.

**Comment by greg0ire** [comment]
I see you are having a hard time. Here is another solution for you : 

```
git fetch --all
git reset --hard 139d110
git cherry-pick 578eb50
git push --force
```

**Comment by axzx** [comment]
sorry,
I've created a new one PR.

**Comment by greg0ire** [comment]
Ok fine :)

**Pull request closed without merging**